Homework_8.py

```diff
@@ -0,0 +1,73 @@
+import json  # import module for work with json
+from Homework_7 import *  # import previous homeworks (5-6)
+
+
+class Parse:
```

Comment:
Why the name is Parse?:) It is quite confusing:) Maybe add some certain details to the name like in previous HWs? Smth like Publish_json or maybe better JSONPublisher